src/main/java/com/softserveinc/Task01.java

```diff
@@ -16,9 +16,197 @@
  */
 public class Task01 {
     public static void main(String[] args) {
-                              //  I  | II | III | IV |  V  |  VI | VII | VIII | IX  |  X  | XI |  XII
-        double[] temperature = { -1.7, -1.0, 2.6,  9.0, 15.1, 19.4, 21.4, 21.2, 17.1, 11.1, 5.9,  1.4 };
+        //  I  | II | III | IV |  V  |  VI | VII | VIII | IX  |  X  | XI |  XII
+        double[] temperature = {-1.7, -1.0, 2.6, 9.0, 15.1, 19.4, 21.4, 21.2, 17.1, 11.1, 5.9, 1.4};
         // TODO: Write your code here
+        double sum = 0;
+
+        for (double monthtemperature : temperature) {
+            sum += monthtemperature;
+        }
+        System.out.println("Average annual temperature: " + sum / temperature.length);
+
+        double min = temperature[0];
+        double max = temperature[0];
+
+        for (double month : temperature) {
```

Comment:
Ues for loop with indexes to find the month with the highest and the lowest temperature in the year.

Reply:
Thanks for the feedback. Resolved.